components/unit/Pagination.vue

```diff
@@ -1,46 +1,44 @@
 <template>
-  <div v-if="!hidePagination" class="unit-pagination">
+  <div class="unit-pagination">
     <BasicLink
       v-if="previousUnit"
-      :to="`/course/${activeCourseSlug}/${previousUnit.slug}`"
+      :to="`/course/${activeCourse?.slug}/${previousUnit.slug}`"
       as-button
       button-kind="secondary"
-      @click="setActiveUnitIdBySlug(previousUnit.slug)"
     >
       {{ previousUnit.title }}
     </BasicLink>
     <BasicLink
       v-if="nextUnit"
-      :to="`/course/${activeCourseSlug}/${nextUnit.slug}`"
+      :to="`/course/${activeCourse?.slug}/${nextUnit.slug}`"
```

Comment:
```suggestion
      :to="`/course/${activeCourse!.slug}/${nextUnit.slug}`"
```

I think we should assume that the unit exits (with the `!` instead of `?`).

And if not, we should control what to render if there is no course.
But I believe that it cannot be the case that there is no course here.

Reply:
Done e290196d56aa52355886819347e4046508ae99dd